src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -181,7 +181,25 @@ ExportQueryExecutionTrees::idToStringAndTypeForEncodedValue(Id id) {
   }
 }
 
-// ___________________________________________________________________________
+// _____________________________________________________________________________
+ad_utility::triple_component::LiteralOrIri
+ExportQueryExecutionTrees::getLiteralOrIriFromVocabIndex(
+    const Index& index, Id id, const LocalVocab& localVocab) {
+  using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
+  std::optional<std::string> entity;
+  switch (id.getDatatype()) {
+    case Datatype::LocalVocabIndex:
+      return localVocab.getWord(id.getLocalVocabIndex());
+    case Datatype::VocabIndex:
+      entity = index.idToOptionalString(id.getVocabIndex());
```

Comment:
```suggestion
     auto entity = index.idToOptionalString(id.getVocabIndex());
```
and remove the definition in line 189.